src/components/EventCard.tsx

```diff
@@ -52,7 +52,17 @@ const EventCard: React.FC<Props> = ({ event }: Props) => {
                   />
                 </Link>
               </span>
-              <span>{event.title}</span>
+              {(event.published ?? 'true') == 'false' ? (
```

Comment:
for shortness and to avoid code duplication, you could use the function `isPublished`
```suggestion
              {isPublished(event) ? (
```